build.gradle

```diff
@@ -738,48 +741,52 @@ jlink {
     // This is based on ./gradlew suggestMergedModuleInfo, sort, strip ";"", comment non-used modules, and include the suggested directives here.
     //   However, we need to fine-tune this manually (non-alphabetic order)
     mergedModule {
-        requires 'javafx.base'
```

Comment:
For the xth time! Do not reorder the dependencies here!

Reply:
see the comment above in the build.gradle!

Reply:
I wanted to find out the reason, while I am on it. We did not document it back than. We just said: Some local experiments showd that it does not work.

I tried today with alphabetic ordering. It works. - Having alphabetic ordering in place helps to include the output of `./gradlew suggestMergedModuleInfo`. Otherwise, it is an endless brainfuck.

If it does not, we have enough time before a release.